src/torchcodec/_core/CudaDeviceInterface.cpp

```diff
@@ -426,35 +426,36 @@ void CudaDeviceInterface::convertAVFrameToFrameOutput(
     dst = allocateEmptyHWCTensor(frameDims, device_);
   }
 
```

Comment:
Note for below: that's introducing yet another dependency relationship between the BETA interface and the default interface. All of this badness is resolved in https://github.com/meta-pytorch/torchcodec/pull/931